gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/web/publish/PluginPublishController.kt

```diff
@@ -92,15 +93,15 @@ class PluginPublishController(
             .addFormDataPart(
               "plugin",
               format("%s-%s.zip", pluginId, pluginVersion),
-              RequestBody.create(MediaType.parse("application/octet-stream"), body)
+              RequestBody.create(("application/octet-stream").toMediaType(), body)
```

Comment:
I get a deprecation warning with this.  This
```suggestion
body.toRequestBody(("application/octet-stream").toMediaType())
```
along with
```
import okhttp3.RequestBody.Companion.toRequestBody
```
makes the warning go away.